detection_rules/devtools.py

```diff
@@ -147,6 +152,43 @@ def build_integration_docs(ctx: click.Context, registry_version: str, pre: str,
     return docs
 
 
+@dev_group.command("bump-versions")
```

Comment:
A couple of additional thoughts. When we release or build packages, prior to 8.7 it would be a Kibana package and Integrations package. As of 8.7+ it will only be an integrations package which relies solely on the registry data. The details and changes of this have not begun to take affect yet, nor should they for this PR.

However, `version` will typically not match `package.name` nor should it match `8.7.0`. For example, if we are working on the 8.7 release, we would use this command to bump minor where `package.name`, `kibana.version` and `version` would be 8.7 for the "Prep for next release PR". When we build the packages however, `version` would then be bumped a patch.